zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -216,54 +226,75 @@ CREATE TABLE orchard_received_note_spends (
     UNIQUE (orchard_received_note_id, transaction_id)
 )";
 
-/// Stores the current UTXO set for the wallet, as well as any transparent outputs
-/// previously observed by the wallet.
+/// Stores the transparent outputs received by the wallet.
 ///
 /// Originally this table only stored the current UTXO set (as of latest refresh), and the
 /// table was cleared prior to loading in the latest UTXO set. We now upsert instead of
 /// insert into the database, meaning that spent outputs are left in the database. This
-/// makes it similar to the `*_received_notes` tables in that it can store history, but
-/// has several downsides:
-/// - The table has incomplete contents for recovered-from-seed wallets.
-/// - The table can have inconsistent contents for seeds loaded into multiple wallets
+/// makes it similar to the `*_received_notes` tables in that it can store history.
```

Comment:
```suggestion
/// Originally this table (when it was named `utxos`) only stored the current UTXO set as
/// of the latest refresh, and the table was cleared prior to loading in the latest UTXO set.
/// We now upsert instead of insert into the database, meaning that spent outputs are left in
/// the database. This makes it similar to the `*_received_notes` tables in that it can store
/// history.
///